vir/simdize.h

```diff
@@ -745,7 +745,7 @@ namespace vir
 	VIR_ALWAYS_INLINE constexpr void
 	_store_elements_via_permute(T* addr, std::integer_sequence<int, Is...>) const
 	{
-#if VIR_HAVE_BUILTIN_SHUFFLEVECTOR
+#if VIR_HAVE_BUILTIN_SHUFFLEVECTOR and __clang_major__ != 17
```

Comment:
Can you please put a short note close by to the reason for this very specific thing? Usually a `<=` or `>=` is obvious as in "not yet supported" or something. But this looks very suspicious. And having to run a `git blame` to find the exact commit and reasoning doesn't sound too appealing.

Reply:
You're right. This was a "does this stop the miscompilations?" test and then after half an hour waiting for the confirmation I didn't go back to it...